### PR TITLE
Implement remote strings for survey

### PR DESCRIPTION
**Pull request by andrews-moc**
[MOB-2565](https://glia.atlassian.net/browse/MOB-2565)

[MOB-2565]: https://glia.atlassian.net/browse/MOB-2565?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Pelkar — CHANGES_REQUESTED** [review]
Please have a look at the comments and also make sure that all the error messages are also setup correctly

File: widgetssdk/src/main/java/com/glia/widgets/survey/viewholder/BooleanQuestionViewHolder.kt [hunk 1]
```diff
@@ -32,8 +35,18 @@ class BooleanQuestionViewHolder(
         booleanTheme?.title?.also(title::applyTextTheme)
 
         val buttonConfiguration = questionStyle.optionButton
+        setupYeButton(buttonConfiguration)
```
[the remark this answers is not in the document]
Comment (gugalo) [reply widgetssdk/src/main/java/com/glia/widgets/survey/viewholder/BooleanQuestionViewHolder.kt:38]: I like the Ye Button )

**Review by gugalo — APPROVED** [review]
I have nothing to add to Karl review

File: widgetssdk/src/main/res/layout/survey_boolean_question_item.xml [hunk 2]
```diff
@@ -24,34 +24,34 @@
...
         android:layout_height="wrap_content"
         android:textColor="@color/glia_system_negative_color"
-        android:text="@string/glia_survey_required_error_message"
         android:layout_marginStart="@dimen/glia_large_x_large"
         android:drawablePadding="@dimen/glia_small"
         app:drawableStartCompat="@drawable/ic_attention"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        tools:text="@string/survey_action_validation_error" />
```
[the remark this answers is not in the document]
Comment (andrews-moc) [reply widgetssdk/src/main/res/layout/survey_boolean_question_item.xml:55]: @Pelkar It is set up in SurveyViewHolder. And `BooleanQuestionViewHolder` inherits `SurveyViewHolder`.

**Review by Pelkar — APPROVED** [review]

**Pull request merged**